container-engine-lib/lib/backend_impls/docker/object_attributes_provider/object_attributes_provider.go

```diff
@@ -17,10 +17,12 @@ import (
 const (
 	engineServerNamePrefix = "kurtosis-engine"
 	logsDatabaseName       = "kurtosis-logs-db"
+	logsAggregatorName     = "kurtosis-logs-aggregator"
 
 	//We always use the same name because we are going to have only one instance of this volume,
 	//so when the engine is restarted it mounts the same volume with the previous logs
-	logsDatabaseVolumeName = logsDatabaseName + "-vol"
+	logsDatabaseVolumeName   = logsDatabaseName + "-vol"
```

Comment:
Very confused by why we still need those DB things here and everywhere. Are you keeping that for a future PR or something?

Reply:
removed